Gems/Multiplayer/Code/Source/Pipeline/NetworkSpawnableHolderComponent.cpp

```diff
@@ -28,18 +30,40 @@ namespace Multiplayer
 
     void NetworkSpawnableHolderComponent::Activate()
     {
+        const auto agentType = GetMultiplayer()->GetAgentType();
+        const bool shouldSpawnNetEntities =
```

Comment:
I should make this a single IsHost() call at some point.. we sure duplicate this check in enough places :\